Refaktorering av grid

antidecaf opened the pull request:
## Beskrivelse

* [Flex](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout)-styling er skrevet om til [grid](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout).
* `ffe-grid`/`<Grid>` har ikke lenger noe luft i topp eller bunn by default. Luft over og under kan i stedet legges til på grid-rader ved hjelp av disse modifierne:
    * `.ffe-grid__row--padding-{size}` / `<GridRow padding={size}>`
    * `.ffe-grid__row--margin-{size}` / `<GridRow margin={size}>`
    * Størrelsene korresponderer med [spacing i ffe](https://design.sparebank1.no/profil/spacing/)
* `InlineGrid` og `condensed`-varianten er fjernet helt til fordel for et sett med modifiere for `gap` 
* `.ffe-grid__row-wrapper` er ikke lenger nødvendig å bruke for å legge bakgrunnsfarge på rader
* En del modifiere er fjernet:
    * Modifere som endrer rekkefølge på kolonner i en rad:
        * `.ffe-grid__row--reverse`
        * `.ffe-grid__row--horizontal`
    * Modifiere som endrer plassering av innhold i hver kolonne:
        * `.ffe-grid__col--reverse`
        * `.ffe-grid__col--horizontal`
        * `.ffe-grid__col--start`
        * `.ffe-grid__col--center`
        * `.ffe-grid__col--end`
        * `.ffe-grid__col--top`
        * `.ffe-grid__col--middle`
        * `.ffe-grid__col--bottom`
        * `.ffe-grid__col--around`
        * `.ffe-grid__col--between`

Foruten modifierne som er fjernet fungerer både api-et i `ffe-grid-react` og styling i `ffe-grid` likt som før.

## Motivasjon og kontekst

Grid-komponenten er kompleks og overveldende å forholde seg til. Den er i utgangspunktet tenkt å være et verktøy for å lage layouter, spesielt på sidenivå, men inneholder i dag en rekke valgmuligheter som tildels går i beina på hvordan grid brukes som verktøy i designprosessen. Stylingen er også skrevet om for å redusere kompleksitet. 

## Testing

Testet lokalt, i Chrome, Firefox og Safari

antidecaf committed:
feat(ffe-grid): omskriving av grid-styling

BREAKING CHANGE: grid-styling er skrevet om, og disse modifierne er fjernet

* .ffe-grid__row--reverse
* .ffe-grid__row--horizontal
* .ffe-grid__col--reverse
* .ffe-grid__col--horizontal
* .ffe-grid__col--start
* .ffe-grid__col--center
* .ffe-grid__col--end
* .ffe-grid__col--top
* .ffe-grid__col--middle
* .ffe-grid__col--bottom
* .ffe-grid__col--around
* .ffe-grid__col--between

antidecaf committed:
feat(ffe-grid-react): omskriving av grid

BREAKING CHANGE: grid-styling er skrevet om, og mange modifiere er fjernet

antidecaf committed:
fix(component-overview): fjerner ubrukt prop

kwltrs reviewed (changes requested):
Ser bra ut :+1: 

Kan du liste opp de fjernete modifiers i commit meldingen til ffe-grid slik at det blir med i CHANGELOG?

antidecaf commented:
> Kan du liste opp de fjernete modifiers i commit meldingen til ffe-grid slik at det blir med i CHANGELOG?

Done.

antidecaf committed:
feat(ffe-grid): fjerner condensed og inline, legger til modifiere for gap og spacing

BREAKING CHANGE:  og  fjernes, og erstattes med modifiere for gap, margin og padding

antidecaf committed:
feat(ffe-grid-react): fjerner condensed og inline, legger til modifiere for gap og spacing

BREAKING CHANGE:  og -modifieren fjernes, og erstattes med modifiere for gap, margin og padding

antidecaf committed:
fix(component-overview): oppdaterte eksempler på grid
